Cargo.toml

```diff
@@ -3,36 +3,37 @@ name = "nom-derive"
 description = "Custom derive nom parsers from struct"
 version = "0.10.0"
 license = "MIT/Apache-2.0"
-keywords = ["parser","nom"]
+keywords = ["parser", "nom"]
 categories = ["parsing"]
 authors = ["Pierre Chifflier <chifflier@wzdftpd.net>"]
 homepage = "https://github.com/rust-bakery/nom-derive"
 repository = "https://github.com/rust-bakery/nom-derive.git"
 readme = "README.md"
-edition = "2018"
+edition = "2024"
 
 include = [
-  "LICENSE-*",
-  "CHANGELOG.md",
-  "UPGRADING.md",
-  "README.md",
-  ".gitignore",
-  ".travis.yml",
-  "Cargo.toml",
-  "nom-derive-impl/Cargo.toml",
-  "nom-derive-impl/*.rs",
-  "src/*.rs",
-  "src/meta/*.rs",
+    "LICENSE-*",
+    "CHANGELOG.md",
+    "UPGRADING.md",
+    "README.md",
+    ".gitignore",
+    ".travis.yml",
+    "Cargo.toml",
+    "nom-derive-impl/Cargo.toml",
+    "nom-derive-impl/*.rs",
+    "src/*.rs",
+    "src/meta/*.rs",
 ]
 
 [workspace]
 members = ["nom-derive-impl"]
 
 [dependencies]
-nom = { version = "7.0", default-features = false, features = ["std"] }
-nom-derive-impl = { version="=0.10.0", path="./nom-derive-impl" }
+nom = { version = "8.0", default-features = false, features = ["std"] }
+nom-derive-impl = { version = "=0.10.0", path = "./nom-derive-impl" }
```

Comment:
This version number should be incremented (as well as in `nom-derive-impl/Cargo.toml`), because crates will not be compatible